inf-clojure.el

```diff
@@ -1220,7 +1220,7 @@ See variable `inf-clojure-buffer'."
       (inf-clojure--read-or-nil)
       (inf-clojure--list-or-nil))))
 
-(defconst inf-clojure-clojure-expr-break-chars " \t\n\"\'`><,;|&{(")
+(defconst inf-clojure-clojure-expr-break-chars " \t\n\"\'`><,;|&{()[]1234567890")
```

Comment:
Seems to me the list of all digits should be replaced with something like `[0-9]` or `[:digit]`. Just check the exact syntax for this.